exporter/googlemanagedprometheusexporter/README.md

```diff
@@ -63,16 +63,6 @@ receivers:
                 - action: labelmap
                 regex: __meta_kubernetes_pod_label_(.+)
 processors:
-    # groupbyattrs promotes labels from metrics to resources, allowing them to
-    # be added to the prometheus_target monitored resource.
-    # This allows exporters which monitor multiple namespaces, such as
-    # kube-state-metrics, to override the namespace in the resource by setting
```

Comment:
We should still describe how to use groupbyattrs (e.g. for KSM), but should document it at the bottom, and should not include it in the default config, as most users copy-paste it.

Reply:
updated, let me know what you think.

Reply:
looks good.